app/templates/releases/index.hbs

```diff
@@ -25,7 +25,29 @@
 
 <h2>Upcoming releases</h2>
 
-<ReleaseTimeline class="hide-on-mobile" @project={{emberBetaProject}} />
+The Ember project aims to release approximately every 6 weeks.
+For a very rough estimate of the timeline for the next release,
+you can look at the date of the most recent
+<a href="https://blog.emberjs.com/tags/releases.html">
+  official release blog post
+</a>
+and count forward by six weeks.
```

Comment:
```suggestion
and count forward by 6 weeks.
```
Minor thing, but it feels better to me to be consistent (using numbers vs spelled-out) when referencing the # of weeks. So we'd either change that here and in Our goals or reverse the references using numbers.